bin/import.ml

```diff
@@ -28,15 +28,20 @@ module Cached_digest = Dune.Cached_digest
 module Profile = Dune.Profile
 include Common.Let_syntax
 
-let make_memory () =
+let make_cache () =
   let handle = function
     | Dune_manager.Client.Dedup (target, source, digest) ->
       Scheduler.send_dedup target source digest
   in
   match Sys.getenv_opt "DUNE_CACHE" with
-  | Some _ ->
-    Fiber.return (Some (Result.ok_exn (Dune_manager.Client.make handle)))
-  | _ -> Fiber.return None
+  | Some v ->
+    let cache = Result.ok_exn (Dune_manager.Client.make handle) in
+    Fiber.return
+      ( if v = "check" then
+        Build_system.Check cache
+      else
+        Build_system.Enabled cache )
```

Comment:
We should be strick about the values we accept here, otherwise we can't detect typos

Reply:
What should we accept ? "1", "true", "enabled" ?

Reply:
I'd say `1` and `check`